secure_message/repository/modifier.py

```diff
@@ -91,12 +91,14 @@ def del_draft(draft_id):
         del_draft_status = "DELETE FROM securemessage.status WHERE msg_id='{0}' AND label='{1}'".format(draft_id, Labels.DRAFT.value)
         del_draft_event = "DELETE FROM securemessage.events WHERE msg_id='{0}'".format(draft_id)
         del_draft_inbox_status = "DELETE FROM securemessage.status WHERE msg_id='{0}' AND label='{1}'".format(draft_id, Labels.DRAFT_INBOX.value)
+        del_actors = "DELETE FROM securemessage.actors where msg_id='{0}'".format(draft_id)
         del_draft_msg = "DELETE FROM securemessage.secure_message WHERE msg_id='{0}'".format(draft_id)
 
         try:
             db.get_engine(app=db.get_app()).execute(del_draft_status)
             db.get_engine(app=db.get_app()).execute(del_draft_inbox_status)
             db.get_engine(app=db.get_app()).execute(del_draft_event)
+            db.get_engine(app=db.get_app()).execute(del_actors)
```

Comment:
could we just call get_engine once?

Reply:
I must admit I am not sure , I followed on from what was there before . I can look at this

Reply:
Think James comment above supersedes this now

Reply:
tech debt ticket added